pkg/cfg/mapstruct_test.go

```diff
@@ -166,6 +167,75 @@ func TestMapStructIO_ReadZeroAndDefaultValues_Unexported(t *testing.T) {
 	assert.NoError(t, err, "there should be no error during reading of zeros and defaults")
 }
 
+func TestMapStruct_ReadBasic(t *testing.T) {
+	fakeTime := clockwork.NewFakeClock().Now()
+
+	type sourceStruct struct {
+		B    bool          `cfg:"b"`
+		D    time.Duration `cfg:"d" default:"1s"`
+		I    int           `cfg:"i" default:"1"`
+		I8   int8          `cfg:"i8" default:"2"`
+		I16  int16         `cfg:"i16" default:"3"`
+		I32  int32         `cfg:"i32" default:"4"`
+		I64  int64         `cfg:"i64" default:"5"`
+		F32  float32       `cfg:"f32" default:"1.1"`
+		F64  float64       `cfg:"f64" default:"1.2"`
+		S    string        `cfg:"s" default:"string"`
+		T    time.Time     `cfg:"t" default:"2020-04-21"`
```

Comment:
Why is this default different than the value given in source, but the rest is the same?

Reply:
dropped the default tags as they are not needed here